packages/amplify-category-api/package.json

```diff
@@ -110,7 +105,14 @@
     "lodash": "^4.17.21",
     "ora": "^4.0.3",
     "rimraf": "^3.0.0",
-    "uuid": "^8.3.2"
+    "uuid": "^8.3.2",
+    "amplify-util-headless-input": "^1.9.5",
+    "amplify-headless-interface": "^1.15.0"
```

Comment:
`amplify-headless-interface` should be a dev dep and `amplify-util-headless-input` should be a peer dep

Reply:
Let me test that, when I'd had amplify-util-headless-input as a peer dep previously I'd had some build errors for types not found from that lib, and when adding to both peer and dev dep still wasn't seeing it work.

Reply:
I've moved these both to peer deps, and added headless-input to dev-deps as well, since we were failing finding the types withouth.